tools/topology/topology2/include/common/data.conf

```diff
@@ -7,6 +7,7 @@
 ##			bytes	"0x3, 0x12,0x1"
 ##		}
 ##
+
```

Comment:
looks like a spurious change. If you can drop this commit @jsarha , we're good to go

Reply:
Oh, tuned here and there an then just assumed the spurious change was intentional and made a separate commit of it. Its now fixed.